src/relay/backend/interpreter.cc

```diff
@@ -382,8 +386,10 @@ class Interpreter : public ExprFunctor<ObjectRef(const Expr& n)>,
 
     // Project out just the function(s) we need.
     IRModule lowered_projected_mod;
-    auto mod_itr = per_target_module_.find(target->str());
-    ICHECK(mod_itr != per_target_module_.end())
+    std::unordered_map<Target, IRModule, TargetStrHash, TargetStrEqual> per_target_module_std_map_ =
```

Comment:
nit: don't append a _ for local vars since the convention is it indicates a member var.

Reply:
done!